code/script.js

```diff
@@ -1,13 +1,55 @@
-// All the DOM selectors stored as short variables
 const chat = document.getElementById('chat')
+const input = document.getElementById('user-input')
+const form = document.getElementById('name-form')
+const factButtons = document.getElementById('fact-btn')
+const yesNoButtons = document.getElementById('yes-no-btn')
+const yesButton = document.getElementById('yes')
+const noButton = document.getElementById('no')
+const timeOutTime = 1000
+//this makes the setTimeout a constant so we don't have to change the time for each bot response
+//instead we can just change up here and all of them will be changed.
+
 
 // Global variables, if you need any, declared here
+let questionNumber = 0;
 
 // Functions declared here
 
-// This function will add a chat bubble in the correct place based on who the sender is
+//This fuction sets the order of the questions with a conditonal.
+// Basically sets a order of the questions so the bot
+//does not repeat the last question over and over again. It makes it move on to the next.
+//So if we're on the greeting and the user puts in a input it moves on to next question.
+//The nextQuestion (question1/2/3) are defined further down in the code.
+
+//We wrapped this if/else with at setTimeout and used the variable timeOutTime.
+
+const nextQuestion = (answer) => {
+  usrInput(answer)
+  setTimeout(() =>{
+    if (questionNumber === 1) {
+      question1(answer)
+      input.value = ''
+    } 
+    else if (questionNumber === 2) {
+      question2(answer)
+      input.value = ''
+    } 
+    else if (questionNumber === 3) {
+      question3(answer)
+      input.value =''
+    } 
+  
+  }, timeOutTime)
+    
+  }
+
+// This function will add a chat bubble in the correct place based on who the sender is.
+
 const showMessage = (message, sender) => {
   if (sender === 'user') {
+   
+    console.log("hello from the user")
```

Comment:
Maybe this could have been removed when you had made sure that the code snippet works.